compiler/quilt/nodes.py

```diff
@@ -124,8 +171,6 @@ def _set(self, path, value, build_dir=''):
         elif isinstance(value, string_types + (bytes,)):
             # bytes -> string for consistency when retrieving metadata
             value = value.decode() if isinstance(value, bytes) else value
-            if os.path.isabs(value):
```

Comment:
We shouldn't allow absolute paths in the package. I think the confusion is that we need to allow absolute paths in build files (build.yml). I think the answer is that nodes built from absolute paths should get a null path or a sensible default relative path.

Reply:
My reasoning:
 * It was decided to permit absolute paths in `core.py`.
 * It was decided that we didn't want to export absolute paths as absolute
 * Absolute paths currently exist in global, and probably in some teams
 * Since the check needs to be done for existing absolute paths on the export side, we may as well do the work there
 * As it is, we just ignore the absolute-ness of paths, and treat it as any other exported path, so `'/etc/profile'` becomes `'etc/profile'`, and `'c:\windows\cmd.exe'` becomes `'windows\cmd.exe'` -- and these files, like any other, are exported to the specified destination, if and only if the target doesn't exist.

Do you still want me to change that?

Reply:
We could also:

 * scan the existing database(s) and remove absolute paths, or
 * do checking and handling on both on package build and package export (preferably with the same code)

Reply:
We can use the same code for checking for abs paths during building and package export, but we probably shouldn't crash if we see an absolute path during export.